player/lua/stats.lua

```diff
@@ -950,19 +950,43 @@ end
 local function add_audio(s)
     local r = mp.get_property_native("audio-params")
     -- in case of e.g. lavfi-complex there can be no input audio, only output
-    if not r then
-        r = mp.get_property_native("audio-out-params")
-    end
+    local ro = mp.get_property_native("audio-out-params") or r
+    r = r or ro
     if not r then
         return
     end
 
+    local merge = function(r, ro, prop)
+        local a = r[prop] or ro[prop]
+        local b = ro[prop] or r[prop]
+        return (a == b or a == nil) and a or (a .. " → " .. b)
+    end
+
     append(s, "", {prefix=o.nl .. o.nl .. "Audio:", nl="", indent=""})
     append_property(s, "audio-codec", {prefix_sep="", nl="", indent=""})
-    local cc = append(s, r["channel-count"], {prefix="Channels:"})
-    append(s, r["format"], {prefix="Format:", nl=cc and "" or o.nl,
+    if append_property(s, "current-ao", {prefix="AO:", nl="",
+                                         indent=o.prefix_sep .. o.prefix_sep}) then
+        append_property(s, "audio-device", {prefix_sep="", prefix="(", suffix=")",
+                                            no_prefix_markup=true, nl="", indent=" "})
+    end
+    local mute = mp.get_property_native("mute") and " (Muted)" or ""
+    local vol = append_property(s, "volume", {prefix="Volume:", suffix="%" .. mute})
+    if mp.get_property_native("volume-gain") > 1e-6 then
+        local gain = append_property(s, "volume-gain", {prefix="Gain:", suffix=" dB",
+                                                        nl=vol and "" or o.nl,
+                                                        indent=vol and o.prefix_sep .. o.prefix_sep})
+    end
+    local ao_mute = mp.get_property_native("ao-mute") and " (Muted)" or ""
+    append_property(s, "ao-volume", {prefix="AO Volume:", suffix="%" .. ao_mute,
+                                     nl=(vol or gain) and "" or o.nl,
+                                     indent=(vol or gain) and o.prefix_sep .. o.prefix_sep})
```

Comment:
I don't know. I added what we can here. At least AO volume is useful to quickly check imho. And it is one line.

Reply:
> AO volume can be placed on the same line as current AO.

Would be too long. It is already long with `Audio: <codec name> AO: (<output name>)` where both names have variable length. I don't see how we can fit more there.

Reply:
I find overhanging `AO` nice and similar to `Context` and `HW` in other sections. Of course it can go in separate line, but it is more like a section title. I don't think it deserves separate line.

2nd line show `volume`, `mute`, `ao-volume`, `ao-mute`, `volume-gain` props in one line, do you want to remove it fully? Because just for `ao-volume` wasting a line is not worth it.

Reply:
```
Audio: <audio-codec> AO: (<current-ao>)
    Device: <audio-device> AO Volume: <ao-volume> (<ao-mute>)
    ...
```
And remove  `volume`, `mute`, `volume-gain`. I guess we can go for that.

Reply:
Done. Frankly not sure how I like separate line for device, but I can get used to it.